File: doc-project-api-java/src/main/java/com/example/docprojectapi0java/FileEntity.java
package com.example.docprojectapi0java;

import javax.annotation.processing.Generated;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Table(name= "files")
@Entity()
public class FileEntity {

    @Id
    @GeneratedValue(strategy = jakarta.persistence.GenerationType.IDENTITY)
    private int id;

    @Column
    private String name;

    @Column
    private int size;

    public FileEntity(String name, int size) {
        this.name = name;
        this.size = size;
    }
}

File: gdsc-doc-project-api/src/entities/file.entity.ts
import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';

@Entity('document_template')
export class FileEntity {
  @PrimaryGeneratedColumn()
  id: number;
  @Column()
  name: string;
  @Column()
  size: number;
  @Column()
  createdAt: Date;
}
